src/PrebidMobile/PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -107,43 +105,44 @@ import ObjectiveC.runtime
     
     // MARK: - DEPRECATED adunit user keywords (user.keywords)
 
-    var userKeywords: [String: Set<String>] {
-        Log.info("user keywords are \(customKeywords)")
-        return customKeywords
-    }
-
     /**
-     * This method obtains the user keyword & value user for targeting
-     * if the key already exists the value will be appended to the list. No duplicates will be added
+     The func uses Targeting.addUserKeyword() inside.
+     - Parameters:
+        - key: parameter is omitted
+        - value: is passed to Targeting.addUserKeyword()
      */
-    @available(*, deprecated, message: "Please use Targeting.shared.addUserKeyword() method instead")
+    @available(*, deprecated, message: "Please use Targeting.addUserKeyword() method instead")
     public func addUserKeyword(key: String, value: String) {
```

Comment:
if key is omitted, why not change the API to be just passing one String, same as Android?

Reply:
yes, the new targeting keywords should not utilize a key. It should be a static string. I believe this may be a mistake.

Reply:
Hello @anwzhang,

>if key is omitted, why not change the API to be just passing one String

I did not do it because it is a public API and I want to have a backward compatibility 

>same as Android

Android version has the same changes
https://github.com/prebid/prebid-mobile-android/blob/67df7e2f9be348f33147681454a1e94f66ec1a97/PrebidMobile/API1.0/src/main/java/org/prebid/mobile/AdUnit.java#L150

If you wish I can change PRs for iOS and Android what will entail breaking changes as a result we have to change a major version